src/newViews/ProjectEdit/GeoAreas/AdminLevelMap/index.tsx

```diff
@@ -0,0 +1,259 @@
+import React, { useMemo, useState, useCallback } from 'react';
```

Comment:
This feels redundant.
If we are not immediately showing preview of uploaded geoarea, we should not even require a separate map component.